htdocs/index.php

```diff
@@ -74,7 +74,16 @@
         (!empty(\icms::$user) && is_object(\icms::$user)) ? \icms::$user->uid : 0
     );
 
-    $icmsConfig['startpage'] = $icmsConfig['startpage'][$group];
+    // added failover to default startpage for the registered users group -- JULIAN EGELSTAFF Apr 3 2017
+	$groups = (!empty(\icms::$user) && is_object(\icms::$user)) ? \icms::$user->getGroups() : array(XOOPS_GROUP_ANONYMOUS);
```

Comment:
@MekDrop if all else checks out, we can merge and then make the changes you suggest.